index.html

```diff
@@ -4,9 +4,222 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
-    <link href="index.css" rel="stylesheet" />
+    <link href="css/index.css" rel="stylesheet" />
   </head>
   <body>
-    <h1>Hello!</h1>
+    <div class="container">
+      <header class="header">
+        <h1 class="text-center" id="title">freeCodeCamp Survey Form</h1>
+        <p class="description text-center" id="description">
+          Thank you for taking the time to help us improve the platform
+        </p>
+      </header>
+      <form id="survey-form">
+        <div class="form-group">
+          <label for="name">Name</label>
+          <input
+            class="form-control"
+            id="name"
+            name="name"
+            type="text"
+            aria-label="name"
+            placeholder="Enter your name"
+            required
+          />
+        </div>
+        <div class="form-group">
+          <label for="email">Email</label>
+          <input
+            class="form-control"
+            id="email"
+            name="email"
+            type="email"
+            aria-label="email"
+            placeholder="Enter your Email"
+            required
+          />
+        </div>
+        <div class="form-group">
+          <label for="number">Age<span class="clue">(optional)</span></label>
+          <input
+            class="form-control"
+            id="number"
+            name="age"
+            type="number"
+            aria-label="number"
+            min="10"
+            max="99"
+            placeholder="Age"
+          />
+        </div>
+        <div class="form-group">
+          <p>Which option best describes your current role?</p>
+          <select
+            class="form-control"
+            id="dropdown"
+            name="role"
+            type=""
+            aria-label="dropdown"
+            min=""
+            max=""
+            required
+          >
+            <option value="student" disabled="" selected="">Select current role</option>
+            <option value="student" disabled="" selected="">Student</option>
+            <option value="job">Full Time Job</option>
+            <option value="learner">Full Time Learner</option>
+            <option value="preferNo">Prefer not to say</option>
+            <option value="other">Other</option>
+          </select>
+        </div>
+
+        <div class="form-group">
+          <p>Would you recommend freeCodeCamp to a friend?</p>
+          <label for="radio"></label>
+          <input
+            class="input-radio"
+            id="radio"
+            name="user-recommend"
+            type="radio"
+            value="definitely"
+            aria-label="radio"
+            checked
+          />Definitely
```

Comment:
Looks like the `label` and `input` element are not properly associated. Also, you wont need the `aria-label` if they are well associated.

Reply:
I havent taken note of that and will work on it